Packages/com.unity.inputsystem/InputSystem/API/Input.actions.cs

```diff
@@ -0,0 +1,534 @@
+        {
+            Debug.Assert(InputSystem.actions != null, "Global actions have not been correctly initialized");
+
+            var action = InputSystem.actions?.FindAction(string.IsNullOrEmpty(actionMapName)
+                ? actionName
+                : $"{actionMapName}/{actionName}");
+            return action != null && action.IsPressed();
+        }
+
+        /// <summary>
+        /// True in the frame that the action started.
+        /// </summary>
+        /// <param name="actionMapName"></param>
+        /// <param name="actionName"></param>
+        /// <returns></returns>
+        public static bool WasPressedThisFrame(string actionName, string actionMapName = "")
+        {
+            Debug.Assert(InputSystem.actions != null, "Global actions have not been correctly initialized");
+
+            var action = InputSystem.actions?.FindAction(string.IsNullOrEmpty(actionMapName)
+                ? actionName
+                : $"{actionMapName}/{actionName}");
+            return action != null && action.WasPressedThisFrame();
+        }
+
+        /// <summary>
+        /// True in the frame that the action ended.
+        /// </summary>
+        /// <param name="actionMapName"></param>
+        /// <param name="actionName"></param>
+        /// <returns></returns>
+        public static bool WasReleasedThisFrame(string actionName, string actionMapName = "")
+        {
+            Debug.Assert(InputSystem.actions != null, "Global actions have not been correctly initialized");
+
+            var action = InputSystem.actions?.FindAction(string.IsNullOrEmpty(actionMapName)
+                ? actionName
+                : $"{actionMapName}/{actionName}");
+            return action != null && action.WasReleasedThisFrame();
+        }
+
+        public static bool IsPressed<TActionType>(Input<TActionType> input) where TActionType : struct
+        {
+            return input.isPressed;
+        }
+
+        public static bool WasPressedThisFrame<TActionType>(Input<TActionType> input) where TActionType : struct
+        {
+            return input.wasPressedThisFrame;
+        }
+
+        public static bool WasReleasedThisFrame<TActionType>(Input<TActionType> input) where TActionType : struct
+        {
+            return input.wasReleasedThisFrame;
+        }
+
+        internal static void InitializeGlobalActions(string defaultAssetPath = null, string assetPath = null)
+        {
+#if UNITY_EDITOR
+            if (!EditorApplication.isPlayingOrWillChangePlaymode)
+                return;
+
+            InputSystem.actions = GlobalActionsAsset.GetOrCreateGlobalActionsAsset(assetPath, defaultAssetPath);
+#else
+            // find the source generated global actions runtime file in the users assembly and use it to load the
+            // runtime actions
+            var runtimeAssetLoaderType = AppDomain.CurrentDomain.GetAssemblies().SelectMany(a => a.GetExportedTypes())
```

Comment:
I wonder how CoreCLR will impact this code, not a problem to solve for 0.8.0-pre